### DISCUSSION TITLE
Original Tutorial Map?

### DISCUSSION THREAD

**Discussion by Branpanman** [ihhub/fheroes2#4174] [Q&A] [answered]
Out of morbid curiosity, I wanted to know if there's a way to play the original HoMM2 tutorial map in FHeroes2? (what can I say, I'm an old fart who is a sucker for the "Burma Shave" sign joke at the beginning)

I've located the tutorial map save file in my version of HoMM2 Gold I bought on GoG, but I'm not sure if there's a way to add it to the FHeroes2 save files? Let me know if anyone knows how to do this.

**Comment by ihhub** [comment, marked as answer]
Hi @Branpanman , fheroes2 does **not** support saves from the original game. It is done for simplification of development and from licensing point of view. The tutorial map is actually a save file. That's why you can't see it in fheroes2.

**Reply by Branpanman** [reply]
Got it. Thank you for the super quick response, Ihar!

**Comment by LeHerosInconnu** [comment]
Hello @Branpanman,

So you like shaving cream? :D

The Heroes 2 tutorial map has been recreated by "dudejo".
This map named "Tutorial" is marked as version 0.8.
Here is the copy of the text from the file "Tutorial.txt" provided with the scenario:

   ____  __  __   _  __  ___    __   __
  /  _/ / /_/ /  / |/ / / . |  /. \ / /
 /  /_ / __  /  / _  / / _  | / _ / \ \
 |___//_/ /_/  /_//_/ /_/ |_|/_/    /_/
    HTTP://WWW.CELESTIALHEAVENS.COM                          

-------------------
TEST MAP SUBMISSION
-------------------

Map Name:      Tutorial.MP2
Author:        dudejo
E-mail Address: joebatard@hotmail.com
Version:       0.8

Map Type:      Single Player, 1 Human and 2 AI
Map Size:      Small
Game Version:  Heroes 2 Gold
Language:      English

Special Win:   Blue Vs Others
Special Loss:  NONE
Special Other: NONE

Description:   The Tutorial map from Heroes 2. Normally, you can only access it
	       from the saved game. I rebuilt the map from screenshots.

Additional Info: From personal experience with the map, it was clearly designed
		 to be played with Knight and Barbarian as there is only a crystal
 	 	 mine and no other resources can be added without significant
		 modifications to the map itself.

You can find this scenario (and many others) in the "MYP-H2-180913" archive here: https://heroes2.forumactif.com/t474-the-monster-yotta-pack-aka-the-myp-from-the-foe.

Have fun!!! :)

**Reply by Branpanman** [reply]
It's not so much that I like shaving cream, as I'm a dad who is a sucker for bad puns... "Burma Shave" did this better than anyone!

And thank you so much for providing this! I'm looking forward to checking out the rebuilt tutorial map... shaving cream in hand :)

**Reply by Branpanman** [reply]
@LeHerosInconnu bit of a n00b question but when I l click the [URL](https://heroes2.forumactif.com/t474-the-monster-yotta-pack-aka-the-myp-from-the-foe) you provided, where do I actually find the file to download?

**Reply by Branpanman** [reply]
@LeHerosInconnu nevermind! I figured it out... I had to register for the forums 😁

**Reply by Branpanman** [reply]
Thanks for the help @LeHerosInconnu ! I was able to load the maps and I found the Burma Shave joke inside the Tutorial Map using FHeroes2! My inner child is over the moon!!!

![burma shave FHeroes2](https://user-images.githubusercontent.com/90151207/132385163-05b9600c-d6eb-41bc-b792-071020172d86.png)
